support/troubleshooting/troubleshooting-operating-system-issues.adoc

```diff
@@ -10,3 +10,42 @@ toc::[]
 
 // Investigating kernel crashes
 include::modules/investigating-kernel-crashes.adoc[leveloffset=+1]
+
+include::modules/troubleshooting-enabling-kdump.adoc[leveloffset=+2]
+
+include::modules/troubleshooting-enabling-kdump-day-one.adoc[leveloffset=+2]
+
+=== Testing the kdump configuration
+
+ifdef::openshift-enterprise[]
+See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/configuring-kdump-on-the-command-line_managing-monitoring-and-updating-the-kernel#testing-the-kdump-configuration_configuring-kdump-on-the-command-line[Testing the kdump configuration] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+ifdef::openshift-origin[]
+See the link:https://fedoraproject.org/wiki/How_to_use_kdump_to_debug_kernel_crashes#Step_2:_Capturing_the_Dump[Capturing the Dump] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+=== Analyzing a core dump
```

Comment:
Needs an ID: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings